### ISSUE TITLE
AC node ignoring payloads - BUG

### ISSUE THREAD

**Issue by patrik-malina** [andrei-tatar/node-red-contrib-smartnora#67]
Idk if something changed, but ac node ignoring payloads. It worked few days and now is not.
If i send payloads to thermostat, its ok. 
this is mine flow:

`[{"id":"c1522304.51632","type":"poll-state","z":"18b1d6aa.1b13c9","name":"Midea AC","server":"705207ff.d4d2f8","version":1,"exposeToHomeAssistant":false,"haConfig":[{"property":"name","value":""},{"property":"icon","value":""}],"updateinterval":"1","updateIntervalUnits":"minutes","outputinitially":true,"outputonchanged":true,"entity_id":"climate.midea_ac","state_type":"str","halt_if":"","halt_if_type":"str","halt_if_compare":"is","outputs":1,"x":80,"y":1600,"wires":[["5af309d.108bcf8"]]},{"id":"5af309d.108bcf8","type":"function","z":"18b1d6aa.1b13c9","name":"","func":"var newMsg = { payload: msg.payload };\nnewMsg = {};\nnewMsg.payload = {};\n\nnewMsg.payload.mode = msg.data.state;\n\n\nnewMsg.payload.temperature = msg.data.attributes.current_temperature;\nnewMsg.payload.setpoint = msg.data.attributes.temperature;\nnewMsg.payload.speed = msg.data.attributes.fan_mode;\n\n\nreturn newMsg;\n","outputs":1,"noerr":0,"initialize":"","finalize":"","libs":[],"x":240,"y":1600,"wires":[["4288b054.d5aaf"]]},{"id":"359f27b7.eed6f8","type":"api-call-service","z":"18b1d6aa.1b13c9","name":"mode","server":"705207ff.d4d2f8","version":3,"debugenabled":false,"service_domain":"climate","service":"set_hvac_mode","entityId":"climate.midea_ac","data":"{\"hvac_mode\":\"{{payload.mode}}\"}","dataType":"json","mergecontext":"","mustacheAltTags":false,"outputProperties":[],"queue":"none","x":590,"y":1620,"wires":[[]]},{"id":"c9fa2735.3e8638","type":"poll-state","z":"18b1d6aa.1b13c9","name":"Humidity","server":"705207ff.d4d2f8","version":1,"exposeToHomeAssistant":false,"haConfig":[{"property":"name","value":""},{"property":"icon","value":""}],"updateinterval":"1","updateIntervalUnits":"minutes","outputinitially":true,"outputonchanged":true,"entity_id":"sensor.xiaomi_sensor_humidity","state_type":"num","halt_if":"","halt_if_type":"str","halt_if_compare":"is","outputs":1,"x":80,"y":1660,"wires":[["d3e9f339.fc972"]]},{"id":"d3e9f339.fc972","type":"function","z":"18b1d6aa.1b13c9","name":"","func":"var newMsg = { payload: msg.payload };\nnewMsg = {};\nnewMsg.payload = {};\n\nnewMsg.payload.humidity = Math.round(msg.payload);\n\nreturn newMsg;\n","outputs":1,"noerr":0,"initialize":"","finalize":"","libs":[],"x":240,"y":1660,"wires":[["4288b054.d5aaf"]]},{"id":"4288b054.d5aaf","type":"noraf-ac","z":"18b1d6aa.1b13c9","devicename":"Midea AC","roomhint":"1st Floor","name":"","modes":"off,heat,cool,on,auto,fan-only,dry","unit":"C","rangeMin":"17","rangeMax":"30","topic":"","passthru":false,"bufferRangeCelsius":2,"nora":"d3f45eb.318cca","twofactor":"ack","twofactorpin":"","filter":false,"percentcontrol":false,"language":"en","speeds":[{"n":"Low","v":"low"},{"n":"Medium","v":"medium"},{"n":"High","v":"high"},{"n":"Auto","v":"auto"}],"x":420,"y":1620,"wires":[["359f27b7.eed6f8"]]},{"id":"705207ff.d4d2f8","type":"server","name":"Home Assistant","version":1,"addon":true,"rejectUnauthorizedCerts":true,"ha_boolean":"y|yes|true|on|home|open","connectionDelay":true,"cacheJson":true},{"id":"d3f45eb.318cca","type":"noraf-config","name":"Home","group":"","twofactor":"off","twofactorpin":"","localexecution":true,"structure":""}]`

![Screenshot (127)_LI](https://user-images.githubusercontent.com/73561855/127737026-f39d2a75-80c2-4059-b058-4fa434a06cd8.jpg)

**Comment by storm1er** [comment]
same here but with another message
![image](https://user-images.githubusercontent.com/9821946/127740475-a3d79b06-91b4-4309-9a01-0062537c5b70.png)

**Comment by storm1er** [comment]
Bug disappear after replacing the device with a new one which includes "on" in supported modes.

**Comment by patrik-malina** [comment]
Hm, will try...

**Comment by patrik-malina** [comment]
> same here but with another message
> ![image](https://user-images.githubusercontent.com/9821946/127740475-a3d79b06-91b4-4309-9a01-0062537c5b70.png)

I think because you have thermostatMode instead mode property

**Comment by storm1er** [comment]
@patrik-malina I had this also, adding "on" didn't work, but creating a new device (removing the old one then create another) with the "on" mode made the error vanish :/

**Comment by patrik-malina** [comment]
> @patrik-malina I had this also, adding "on" didn't work, but creating a new device (removing the old one then create another) with the "on" mode made the error vanish :/

When are you talking about new device, do you talk about new node? With new node, it is same behavior for me :(

**Comment by andrei-tatar** [comment]
Found the issue. Will push the fix later today

**Comment by andrei-tatar** [comment]
@patrik-malina Published v1.0.13 and this bug should be fixed. Can you have a look?

**Comment by patrik-malina** [comment]
> @patrik-malina Published v1.0.13 and this bug should be fixed. Can you have a look?

how can i force update? stil not showing new version...

**Comment by andrei-tatar** [comment]
Should be as simple as:

```
cd ~/.node-red
npm install node-red-contrib-smartnora@latest
```
or to get a specific version:
```
npm install node-red-contrib-smartnora@1.0.13
```
